test/commons/aria/get-element-unallowed-roles.js

```diff
@@ -66,16 +67,28 @@ describe('aria.getElementUnallowedRoles', function() {
     var role = 'textbox';
     node.setAttribute('role', role);
     flatTreeSetup(node);
-    var actual = axe.commons.aria.getElementUnallowedRoles(node, true);
+    var actual = getElementUnallowedRoles(node, true);
     assert.isEmpty(actual);
   });
 
   it('returns false with implicit role of row for TR when allowImplicit is set to false via options', function() {
     var node = document.createElement('tr');
     node.setAttribute('role', 'row');
     flatTreeSetup(node);
-    var actual = axe.commons.aria.getElementUnallowedRoles(node, false);
+    var actual = getElementUnallowedRoles(node, false);
     assert.isNotEmpty(actual);
     assert.include(actual, 'row');
   });
+
+  it('works with SerialVirtualNode', function() {
```

Comment:
Can you add at least one additional test showing it doesn't always return empty?